interview-mr/src/main/java/com/sovrn/interview/mr/AverageScore.java

```diff
@@ -32,26 +35,40 @@
  */
 public class AverageScore {
     public static void main(String[] args) throws Exception {
+        System.exit(runMapReduce(args[0], args[1]) ? 0 : 1);
+    }
+
+    // new entry point for testability
```

Comment:
It's impossible to test a method if it calls `System.exit()` so I added a new public method here.